packages/core/src/App/Containers/AccountTypesModal/account-types-modal.jsx

```diff
@@ -31,8 +32,14 @@ const Box = ({ title, description, footer_text, icons, cards }) => {
         <div className='account-types__box'>
             <div className='account-types__box-left'>
                 <h2 className='account-types__box-title'>{title}</h2>
-                <p className='account-types__box-description'>{description}</p>
-                {footer_text && <p className='account-types__box-footer'>{footer_text}</p>}
+                <Text as='p' size='xs' lineHeight='x' className='account-types__box-description'>
```

Comment:
Any reason we don't follow our naming convention here? (**lineHeight**)

Reply:
I followed the convention like className :D ... What should it be then?

Reply:
It's an exception :)) Since `class` is a JavaScript keyword we can't use it outside of its context.
Feel free to rename it to `line_height`.

Reply:
I fixed it on previous PR. #2030